Kconfig.sof

```diff
@@ -109,6 +109,10 @@ config XT_INTERRUPT_LEVEL_5
 	  Select if the platform supports any interrupts of level 5.
 	  Disabling this option allows for less memory consumption.
 
+config COMPILER_WORKAROUND_CACHE_ATTR
+	bool
```

Comment:
We should say which compiler, version and O level here.

Reply:
Done